lib/handlers/receiver_handler.rb

```diff
@@ -138,9 +140,12 @@ def do_process_reply_to(message)
       # If sender for actual reply-to address does not exist
       unless @senders.include?(message.reply_to)
         # Create new sender for reply-to address
-        @senders[message.reply_to] = @receiver.connection.open_sender(
-          message.reply_to
-        )
+        @senders[message.reply_to] = @receiver.connection.open_sender({
+          # Set target address
+          :target => message.reply_to,
+          # Set auto settle
+          :auto_settle => @auto_settle_off ? false : true,
```

Comment:
ack, I will modify the --recv-selector PR to put default value in the dict if nothing is specified on cli

Reply:
I'd do `not @auto_settle_off`, matter of taste, I guess.

Reply:
Yes, it is shorter, I will think about it.